README_cmake.md

```diff
@@ -284,6 +634,16 @@ same platform/bits/os triple (eg. `arm-32-linux`). Features that are in common
 among all targets, including device libraries (like `cuda`) should go in
 `FEATURES`.
 
+Every element of `TARGETS` must begin with the same `arch-bits-os` triple. This
+function understands two _meta-triples_, `host` and `cmake`. The meta-triple
+`host` is equal to the `arch-bits-os` triple used to compile Halide. On
```

Comment:
This bit concerns me. The host architecture used to compile Halide-the-compiler is irrelevant. E.g. you could have built Halide on a machine without avx512 and be using it on a machine with avx512.

Reply:
As I mentioned above, it isn't because it changes the meaning of the "host" meta-target. Of course, the OS and architecture have to remain fixed, but the bitness doesn't.